src/site/components/events_list.drupal.liquid

```diff
@@ -30,5 +33,23 @@
     </div>
   </div>
   {% endif %}
+  {% endif %}
   {% endfor %}
+  {% if count == 0 %}
+  <div class="vads-u-margin-bottom--2">
+    <div class="va-introtext">
+      <p>
+        VA proudly works alongside others to connect Veterans to the benefits
+        they’ve
+        earned. Explore resources for our outreach partners such as Veteran
+        Service
```

Comment:
What do you think of collapsing these single-word lines? `prettier` would normally do this for us automagically, but this is `.liquid`. :man_shrugging:

Reply:
Sure, @cvalarida :+1: